json-schemas/workflowRunsAggregateRequest.json

```diff
@@ -0,0 +1,130 @@
+{
+    "type": "object",
+    "properties": {
+        "workflowRunIds": {
```

Comment:
I was guessing it would something like `where { id { _in } }` but totally ok with keeping it like this for now since the API hasn't actually been released 😄